pom.xml

```diff
@@ -121,6 +126,22 @@
             <version>2.11.2</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>git-client</artifactId>
+            <version>2.7.2</version>
+            <scope>compile</scope>
+        </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
```

Comment:
Use https://plugins.jenkins.io/jackson2-api instead of direct dependencies

Reply:
Intellij is not able to resolve jackson2-api dependency.
![image](https://user-images.githubusercontent.com/19909597/40734969-95abb05e-6457-11e8-9d72-5acf866d49ca.png)

Reply:
This worked with the gourp id ```org.jenkins-ci.plugins```